scripts/aggregate-themes/README.md

```diff
@@ -2,10 +2,9 @@
 
 Terra Toolkit provides a built-in mechanism for aggregating themes.
 
-By default, the terra-toolkit webpack configuration enables theme aggregation when it detects the presence of a `terra-theme.config.js` file.
-
-## Getting started
+By default, the terra-toolkit webpack configuration enables theme aggregation when it detects the presence of a `terra-theme.config.js` file. Theme aggregation supports generating theme files and aggregating existing ones. Generating theme files is the recommended approach. But, for passivity, theme aggregation will aggregate existing theme files first, before falling back to theme generation.
 
```

Comment:
nits: These are just suggestions, if you want to keep what you have that's perfectly fine. 

I wouldn't even mention the differences of strategy in this section. Both strategies aggregate and generate a theme file, the terms here are a little ambiguous.

As a user reads through this I would expect them to implement the first strategy they are presented with, instead immediately being presented with multiple options and passivity concerns. A user looking for documentation on the old strategy can be directed to the section below.

Reply:
Removed. https://github.com/cerner/terra-toolkit/pull/324/commits/0a976941afbe295532f1f4b742f97462fdcfe2a6